README.md

```diff
@@ -166,9 +201,10 @@ To be able to use ccache to speed-up (re)compilation
    ```sh
    pacman -S mingw-w64-i686-ccache 
    ```
-<a name="optional-ccache"/>
+<a name="optional-cmake"/>
```

Comment:
Why does this anchor precede the associated heading when other anchors follow it?

Reply:
Probably because I missed that while reworking the file multiple times. It's still technically valid and won't break, but it's not consistent and thus should change to be consistent across the whole file.